### DISCUSSION TITLE
Star Wars Series

### DISCUSSION THREAD

**Discussion by 0dysseus** [seanap/Plex-Audiobook-Guide#39] [Q&A] [answered]
Hi, thank you very much all your work! So far, everything works great, except for Star Wars books. Many of them are listed as being part of multiple series, like this one: https://www.audible.com/pd/Star-Wars-X-Wing-Mercy-Kill-Audiobook/B008NCT9QI?ipRedirectOverride=true&overrideBaseCountry. When I try to tag it in Mp3Tag using your presets, I only get the first, "Star Wars: Legends" and no series numbers. Is it possible to save the multiple series? Thanks!

**Comment by seanap** [comment, marked as answer]
Star wars has been a pain to tag, unfortunately they require more of a manual approach. As far as multiple series, that is also not supported.  You can create custom tags SERIES2, SERIES-PART2, etc. The problem is no players look for these tags. You can create a collection in plex but you can only have 1 AlbumSort plex tag. Ex. AlbumSort=[universe] [universe-part], [series] [series-part] - [title]

That is also very hard to automate because Audible is not consistent in which series they list first or second.

**Comment by 0dysseus** [comment]
Thanks for the quick answer! I'll edit what I can manually as suggested.
